.evergreen/config/php.ini

```diff
@@ -1 +1,2 @@
 extension=mongodb.so
+memory_limit=-1
```

Comment:
You should increase the limit. Otherwise we don't know how much memory is used if it's too much. What is the memory limit of the job runner?

Reply:
For some reason it aborted when it hit a previous 128M limit, apparently ignoring the 1G limit definer in the runner config. We can also report memory usage from the benchmarks using `perf.send`, which would be a better indicator than failing the build when it hits some limit. Want me to add those numbers?

Reply:
`perf.send` is too late if the process crashes, no?

Reply:
Correct, `perf.send` would not be executed in that case. With `-1` the memory would be unlimited, so the only case in which it would crash is if it used up all memory including the page file, which I'd consider highly unlikely.